magento/magento2
# #26240: Fixed logic for getting option price index for selected swatch option

### Description (*)
Fixed logic for getting option price index for selected swatch option.

### Fixed Issues (if relevant)
1. #26240 

### Manual testing scenarios (*)
1. Enable MAP (Store -> Configuration -> Sales -> Sales -> Minimum Advertised Price -> Enable MAP).

1. Create configurable product for attribute with 'swatch' (Visual Swatch, Text Swatch) type with assoshiated products:
-  simple A: price 1, MAP 11. (ex: 'black' color)
-  simple B: price 1, MAP 12. (ex: 'white' color)

![Expected result](https://user-images.githubusercontent.com/1982263/71675305-25b97480-2d86-11ea-8691-7b5538ce6c21.png)


### Contribution checklist (*)
